source/developers/frontend.markdown

```diff
@@ -39,6 +39,29 @@ Next step is to get the frontend code. When you clone the Home Assistant reposit
 $ script/bootstrap_frontend
 ```
 
+This script will update the git submodule for the polymer project in 
+
+`home-assistant/homeassistant/components/frontend/www_static/home-assistant-polymer`. 
+
+If you're planning on issuing a PR back to the HA codebase you need to fork the polymer project and add your fork as a remote.
+
+```bash
+$ cd homeassistant/components/frontend/www_static/home-assistant-polymer
+$ git remote add <remote name> <github URL to your fork>
+```
+
+When you've made your changes and are ready to push them change to the working directory for the polymer project
+
+``` bash
+$ cd homeassistant/components/frontend/www_static/home-assistant-polymer
+```
+
```

Comment:
I would suggest you combine the two parts into one for readability since the previous section is one part.